app/models/trn_request.rb

```diff
@@ -1,10 +1,16 @@
 # frozen_string_literal: true
 class TrnRequest < ApplicationRecord
-  validates :email, valid_for_notify: true
+  validates :email, valid_for_notify: true, if: %i[itt_provider_answered? email]
```

Comment:
> It seems likely that TrnRequest could make use of a state machine to allow validations to be more declarative based on state.

I see what you mean here.

Another option is creating `FormObject`s that are temporarily instantiated for a particular step in the form. They are instantiated from a `TrnRequest` and saving one writes to a `TrnRequest`, after which it's discarded. The form objects have validations, the `TrnRequest` doesn't.

Another option is to use validation scopes on the `TrnRequest`, and use `valid?(:email)` where appropriate. This is a bit more brittle in my experience.

Reply:
Yeah, some good ideas there. I think seeing more of the whole flow will make this decision clearer. I'll keep waiting.